src/schema/v2/partner.ts

```diff
@@ -179,19 +230,42 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
         resolve: ({ default_profile_id }, _options, { profileLoader }) =>
           profileLoader(default_profile_id).catch(() => null),
       },
-      // TODO: Create a connection for this
-      // shows: {
-      //   type: PartnerShows.type,
-      //   args: omit(PartnerShows.args, "partner_id"),
-      //   resolve: ({ _id }, options) => {
-      //     return PartnerShows.resolve(
-      //       null,
-      //       assign({}, options, {
-      //         partner_id: _id,
-      //       })
-      //     )
-      //   },
-      // },
+      showsConnection: {
+        description: "A connection of shows from a Partner.",
+        type: ShowsConnection.connectionType,
+        args: pageable({
+          sort: {
+            type: ShowSorts,
+          },
+        }),
+        resolve: ({ id }, args, { partnerShowsLoader }) => {
+          const pageOptions = convertConnectionArgsToGravityArgs(args)
+          const { page, size, offset } = pageOptions
+
+          interface GravityArgs {
+            page: number
+            published: boolean
```

Comment:
🍝 `published` isn't a param that winds up getting sent over.

Generally the `GravityArgs` interface used for an upstream request, for a connection, would include `page` `size` and `total_count` to start (since these are needed for proper pagination/scrolling). And then, any of the args that this field accepts.

So this should drop `published`, and also include `sort` (currently that argument isn't getting passed through.